app/components/ui/atoms/PersonPod.js

```diff
@@ -126,18 +133,56 @@ const buildChooserText = ({ role, isRequired, ...props }) => (
   </Flex>
 )
 
-const ChoosePersonPod = ({ onIconClick, ...props }) => {
+const SelectButton = ({ onIconClick, ...props }) => {
   const ChooserText = buildChooserText(props)
   return (
-    <PersonPod
+    <PersonPodContainer
       icon={PlusIcon}
       onIconClick={onIconClick}
       textContainer={ChooserText}
     />
```

Comment:
Rather than passing in `textContainer` as an attribute, you may be able to pass it in as a child

i.e.
```
<PersonPodContainer
  icon={PlusIcon}
  onIconClick={onIconClick}
>
  <ChooserText/>
</PersonPodContainer>
```

`PersonPodContainer` would then use `this.props.children` rather than `this.props.textContainer`.

Reply:
But the `icon` is also a child. So I can't just render all children with `this.props.children`, right?

Reply:
The icon is an attribute, under `this.props.icon`. It shouldn't be in `this.props.children`.

Reply:
The idea is that the component has children as it's content, and has attributes which act like modifiers. It makes it easier to understand.

e.g.
```
<Notification dismissable>
  We use cookies on this website. Do you accept our terms & conditions?
  <Accept/>
  <Decline/>
</Notification>
```

Reply:
I'm not sure that I understand what you mean. Why is `textContainer` any different from `icon`?